composer.json

```diff
@@ -10,20 +10,41 @@
       "homepage": "http://www.auth0.com/"
     }
   ],
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/evansims/jwt.git"
+    },
+    {
+      "type": "git",
+      "url": "https://github.com/php-cache/adapter-common.git"
+    },
+    {
+      "type": "git",
+      "url": "https://github.com/php-cache/hierarchical-cache.git"
+    },
+    {
+      "type": "git",
+      "url": "https://github.com/php-cache/array-adapter.git"
+    }
+  ],
   "require": {
-    "php": "^7.1",
-    "guzzlehttp/guzzle": "~6.0 | ~7.0",
+    "php": "^7.3 | ^8.0",
     "ext-json": "*",
-    "lcobucci/jwt": "^3.3",
+    "ext-openssl": "*",
+    "guzzlehttp/guzzle": "^7.2",
+    "lcobucci/jwt": "dev-3.3-php8-compatibility#daa6aff67a22bf737504143ae55a6913d9ff484d",
```

Comment:
Prehaps we can allow either ^3.3 or ^4.0, as 4.0 is already compatible with php 8.

Reply:
Adapting to 4.0 will require a relatively significant time investment in updating the library, as the upstream author refactored the interface. Will definitely move to 4.0 once we have the bandwidth to do so reliably, but in the meantime this hotfix will give us that PHP 8 compatibility without the overhead.